lib/Command/Room/TRoomCommand.php

```diff
@@ -137,11 +137,11 @@ protected function setRoomPublic(Room $room, bool $public): void {
 
 	/**
 	 * @param Room $room
-	 * @param bool $readOnly
+	 * @param int $readOnly
 	 *
 	 * @throws InvalidArgumentException
 	 */
-	protected function setRoomReadOnly(Room $room, bool $readOnly): void {
+	protected function setRoomReadOnly(Room $room, int $readOnly): void {
```

Comment:
please also change the next line, you need to compare the int value now

Reply:
please recheck the whole function, it needs to be rewritten to handle the int value as it's not a bool any more